src/pages/api/__tests__/version.test.ts

```diff
@@ -0,0 +1,25 @@
+import handler from '../version';
+
+test('version response', async () => {
+  const req = {};
+  const json = jest.fn();
+  const status = jest.fn(() => {
+    return {
+      json,
+    };
+  });
+  const res = {
+    status,
+  };
+
+  handler(req, res);
+  const res_json = json.mock.calls[0][0];
+
+  // console.log(res_json);
```

Comment:
Leftover debug code? It should be removed IMHO.

Reply:
removed